docs/verification.md

```diff
@@ -0,0 +1,201 @@
+# Verification
+
+## Offline Verification
+
+In offline Rekor storage mode Gitsign will store a HashedRekord in Rekor
+corresponding to the commit content.
+
+Unfortunately this is a bit complex to query manually. Roughly this is:
+
+```
+sha256(der(sort(system time | commit data | content type)))
```

Comment:
"commit data" is what, the output of `git cat-file commit <SHA>`?

What's content type? Any why is the time in there if there's already a commit time?

What's the "sort" for?

More generally, what's the motivation for the design here (esp because you could just use the commit SHA)? Does this format come from Git?